storage/blockdevice/COMPONENT_OSPIF/include/OSPIF/OSPIFBlockDevice.h

```diff
@@ -62,6 +62,8 @@
 #define MBED_CONF_OSPIF_OSPI_FREQ 40000000
 #endif
 
+#define MX_FLASH_SUPPORT_RWW
```

Comment:
This is not related to the memory ?

Reply:
Yes, it is. 
I have moved it to MX25LW51245G_config.h.